drivers/hmis/app/models/hmis/ce/match/calculator_factory.rb

```diff
@@ -8,15 +8,20 @@ module CalculatorFactory
     def self.build
       calculator = Dentaku::Calculator.new(case_sensitive: true) # CDED keys are case sensitive, so Dentaku expressions should be too
       calculator.add_function(
-        :includes,
+        :INCLUDES,
```

Comment:
Made these uppercase since the calculator is now case sensitive (See above note)